robottelo/cli/base.py

```diff
@@ -86,6 +92,40 @@ class Base(object):
     logger = logging.getLogger('robottelo')
 
     @classmethod
+    def _handle_response(cls, response, ignore_stderr=None):
+        """Verify ``return_code`` of the CLI command.
+
+        Check for a non-zero return code or any stderr contents.
+
+        :param response: a ``SSHCommandResult`` object, returned by
+         :mod:`robottelo.ssh.command`.
+        :param ignore_stderr: indicates whether to throw a warning in logs if
+         ``stderr`` is not empty.
+        :returns: contents of ``stdout``.
+        :raises robottelo.cli.base.CLIReturnCodeError: If return code is
+         different from zero.
```

Comment:
Can Sphinx parse one-space indentation levels? I'm not sure. Regardless, can you please update to:

```
        :param response: a ``SSHCommandResult`` object, returned by
            :mod:`robottelo.ssh.command`.
        :param ignore_stderr: indicates whether to throw a warning in logs if
            ``stderr`` is not empty.
        :returns: contents of ``stdout``.
        :raises robottelo.cli.base.CLIReturnCodeError: If return code is
            different from zero.
```

Nitpick.